test/helpers/quictestlib.h

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/ssl.h>
+#include <internal/quic_tserver.h>
+
+/* Type to represent the Fault Injector */
+typedef struct ossl_quic_fault OSSL_QUIC_FAULT;
```

Comment:
Given this is test only thing IMO the name should not use the OSSL prefix. I'd use QUIC_TEST or QTEST prefix instead -> QUIC_TEST_FAULT 

Similarly for the structure below (QTEST_ENCRYPTED_EXTENSIONS or something)